File: progetto flutter/lib/models/Lezione.dart
class Lezione {
  Lezione({
    required this.descrizione,
    required this.id,
    required this.titolo,
    required this.url,
  });
  String descrizione;
  String id;
  String titolo;
  String url;
  factory Lezione.fromJson(Map<String, dynamic> json) => Lezione(
    descrizione: json["descrizione"],
    id: json["id"],
    titolo: json["titolo"],
    url: json["url"],
  );
  Map<String, dynamic> toJson() => {
    "descrizione": descrizione,
    "id": id,
    "titolo": titolo,
    "url": url,
  };
}

File: progetto android/app/src/main/java/com/example/progettomobilecamillonitisenigiri/Model/Lezione.kt
package com.example.progettomobilecamillonitisenigiri.Model
/*
* Modello delle lezioni
* */
class Lezione(titolo: String, descrizione: String, url: String, id: String) {
      var titolo: String
      var descrizione: String
      var url: String
      var id: String
    init {
        this.titolo=titolo
        this.descrizione=descrizione
        this.url=url
        this.id=id
    }
    constructor() : this("", "", "", "") {}
}